x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFieldSearch, EuiSpacer } from '@elastic/eui';
+import { TimelineEntry } from './components/timeline_entry';
+import { Immutable, EndpointAction } from '../../../../../../common/endpoint/types';
+
+export const EndpointActivityLog = ({
+  endpointActions,
+}: {
+  endpointActions: Immutable<EndpointAction[]>;
+}) => {
+  return (
+    <>
+      <EuiFieldSearch compressed fullWidth placeholder="Search activity log" />
+      <EuiSpacer size="l" />
+      {endpointActions.map((endpointAction) => (
+        <TimelineEntry key={endpointAction['@timestamp']} endpointAction={endpointAction} />
```

Comment:
is there an `id` on the action? if so, I would suggest using that for the key.

Reply:
2dd22ed92b3c54da9bdcce467dbe2d7e87097e4d